homotopy-web/src/util.rs

```diff
@@ -0,0 +1 @@
+pub mod tree;
```

Comment:
In my rendering branch I've actually moved `tree.rs` into `homotopy-core`. Do you think it's better placed here or there?